packages/ui/components/images/lndr-text/index.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+
+import { View, Image, Text, Dimensions } from 'react-native'
+
+import { applicationName } from 'language'
+
+import style from 'theme/image'
+
+const { height } = Dimensions.get('window');
+const size = height * 0.05;
+
+export default () => (
+  <View>
+    <Image
+      style={{height: size, width: 100}}
+      source={require('images/lndr_text.jpg')}
```

Comment:
No underscores allowed in filenames. Give it a descriptive name like `text-logo` - it's all related to Lndr already no need to include that in the name. We could change the name of the app and would be good to just replace images.